code/datums/actions/xeno_action.dm

```diff
@@ -172,17 +172,35 @@
 
 
 /datum/action/xeno_action/activable
+	///Alternative keybind signal, that will always select the ability, even ignoring keybind flag
+	var/alternate_keybind_signal
```

Comment:
better, make the keybind  signal var a list with keybinds with assoc of w hat is their behaviour, would work better i believe and you could add multiple keybinds on future that way

Reply:
Completly out of the scope this PR.

I'm adding two alternates keybind, i'm not refactoring the whole shit.

"you could add multiple keybinds on future that way" I won't, you won't, no one will. And if anyone will, he'll do it himself